README.md

```diff
@@ -51,24 +63,86 @@ Here is an example of the base app.config.json file
     "Key2": "anotherkey"
   }
 }
+
+```
+
+### Application Configuration Setup
+The App.Config or Web.Config file is the standard configuration file used for .Net Framework applications.  If you only need to use the AppSettings values in your application you can ignore the need for the RockLib.Config.Json file.  The RockLib.Configuration library can pull `AppSetting` values from your App.Config or Web.Config file with no issues.
+
+Here is an example of how to store `AppSetting` values in your app/web.config files
+```
+  <appSettings>
+    <add key="Key1" value="Key1_Value"/>
+  </appSettings>
 ```
 
+### Environment Variables Setup
+There are going to be times when you want to use environment variables inside your application, the RockLib.Configuration library can handle these.
```

Comment:
```diff
- There are going to be times when you want to use environment variables inside your application, the RockLib.Configuration library can handle these.
+ By default, RockLib.Configuration configures itself by reading environment variables. It does so at a higher priority than the RockLib.Config.Json file. This allows some or all of an application's settings to be set via machine, user, or process environment variables. The [Microsoft.Extensions.Configuration documentation](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration#simple-configuration) provides some details on the formatting of environment variables names.
```

Reply:
Works for me, like the link to the ms docs